ern-api-gen/src/languages/JavascriptClientCodegen.ts

```diff
@@ -483,8 +483,8 @@ export default class JavascriptClientCodegen extends DefaultCodegen {
           JavascriptClientCodegen.PROJECT_LICENSE_NAME
         ) == null
       ) {
-        if (info.getLicense() != null) {
-          const license = info.getLicense()
+        const license = info.getLicense()
+        if (license && license.getName()) {
```

Comment:
👀 `getName()` could still pass `undefined` down on L490, so check for it up front